packages/react-scripts/per-lang-loader/index.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+// This file is mostly copied from https://github.com/alienfast/i18next-loader/blob/develop/index.js
+// and adjusted to our needs for per-lang Bundling
+
+const path = require('path');
+const fs = require('fs');
+const globAll = require('glob-all');
+const loaderUtils = require('loader-utils');
+
+function enumerateLangs(dir) {
+  return fs.readdirSync(dir).filter(function(file) {
+    return fs.statSync(path.join(dir, file)).isDirectory();
+  });
+}
+
+//https://github.com/jpillora/node-glob-all#usage
+function findAll(globs, cwd) {
+  const globArray = Array.isArray(globs) ? globs : [globs];
+  return globAll.sync(globArray, { cwd, realpath: true });
+}
+
+const i18nextImport = `
+import i18n from 'i18next'
+import { addTranslations } from "@fs/zion-locale";
+
+const handleError = (locale, namespace) => {
+  console.error(\`failed to load translation - \${locale}" + "\${namespace}"\`)
+}
+const importLocale = (locale, namespace) => {
+  import(\`./\${locale}/\${namespace}\`)
```

Comment:
```suggestion
  /* webpackChunkName: "locales-[request]-[index]" */
  import(\`./\${locale}/\${namespace}\`)
```
if we add a webpack loading hint here, we can get chunk names that distinguish locales from code, which I like
so instead of loading `chunk-987-hash.js`, you get `locales-en-translation-12.js`

Reply:
I like this as well, seems like we'll still need the hash though.